recipes-bsp/u-boot/u-boot-iot2050-2021.x-upstream.inc

```diff
@@ -11,10 +11,36 @@
 require u-boot-iot2050.inc
 
 SRC_URI += " \
-    git://github.com/siemens/u-boot.git;protocol=https;branch=${U_BOOT_BRANCH};rev=${U_BOOT_BRANCH} \
+    git://github.com/u-boot/u-boot.git;protocol=https;branch=${U_BOOT_BRANCH};rev=${U_BOOT_REV} \
```

Comment:
I don't think it's a good idea to use the github fork. Instead we should use our siemens fork or use the gitlab.denx.de directly.

Also considering the git clone speed, better to use the https://gitlab.denx.de/u-boot/u-boot/-/archive/51f65b506f37252acb3cd4184ef5e1fc20da13a2/u-boot-51f65b506f37252acb3cd4184ef5e1fc20da13a2.tar.gz, it's only 20MB in size.

Reply:
Or use https://github.com/siemens/u-boot/archive/51f65b506f37252acb3cd4184ef5e1fc20da13a2.zip, speed is almost the same for me between github and denx.de. But not sure which one is better for somebody else.